test/components/app.spec.js

```diff
@@ -249,7 +249,7 @@ describe('App', () => {
         config: { showsFeedbackButton: true }
       };
 
-      const app = await load('/', { container });
+      const app = await load('/login', { container }).restoreSession(false);
```

Comment:
I am annoyed! This is different from loading '/' with `restoreSession(false)`? I'm pretty sure I had tried that.

Reply:
They should be the same! `restoreSession(false)` is the key part. I just changed it to /login because navigating to / when you're logged out will send you to /login. Might as well go there right away haha.